core/platform/models.py

```diff
@@ -33,6 +33,10 @@
     'question', 'recommendations', 'skill', 'statistics', 'story', 'suggestion',
     'topic', 'user')
 
+NAMES_THAT_CAN_BE_PSEUDONYMIZED = utils.create_enum(  # pylint: disable=invalid-name
```

Comment:
On why `# pylint: disable=invalid-name` is needed please refer to https://github.com/oppia/oppia/pull/7351#discussion_r312731662

Reply:
As mentioned in that other PR, please add a comment here to explain it, for the benefit of future developers.

Reply:
Already done.

Reply:
Sorry ... where? I don't see it in the code.

Reply:
Ah sorry, scratch that. I think I was looking at an older commit.